src/Tests/Microsoft.DotNet.PackageValidation.Tests/ValidatePackageTargetTests.cs

```diff
@@ -83,6 +83,26 @@ public void ValidatePackageTargetRunsSuccessfullyWithBaselineVersion()
             Assert.Equal(0, result.ExitCode);
         }
 
+        [Fact]
+        public void ValidatePackageTargetFailsWithBaselineVersion()
+        {
+            var testAsset = _testAssetsManager
+                .CopyTestAsset("PackageValidationTestProject", allowCopyIfPresent: true)
+                .WithSource();
+
+            var result = new PackCommand(Log, Path.Combine(testAsset.TestRoot, "PackageValidationTestProject.csproj"))
+                .Execute($"-p:PackageOutputPath={testAsset.TestRoot}");
+
+            Assert.Equal(0, result.ExitCode);
+
+            string packageValidationBaselinePath = Path.Combine(testAsset.TestRoot, "PackageValidationTestProject.1.0.0.nupkg");
+            result = new PackCommand(Log, Path.Combine(testAsset.TestRoot, "PackageValidationTestProject.csproj"))
+                .Execute($"-p:PackageVersion=2.0.0;AddBreakingChange=true;PackageValidationBaselinePath={packageValidationBaselinePath}");
+
+            Assert.Equal(1, result.ExitCode);
+            Assert.Contains("error CP0002: Member 'PackageValidationTestProject.Program.SomeApiNotInLatestVersion()' exists on lib/net6.0/PackageValidationTestProject.dll but not on lib/net6.0/PackageValidationTestProject.dll", result.StdOut);
+            Assert.Contains("error CP0002: Member 'PackageValidationTestProject.Program.SomeApiNotInLatestVersion()' exists on lib/netstandard2.0/PackageValidationTestProject.dll but not on lib/netstandard2.0/PackageValidationTestProject.dll", result.StdOut);
```

Comment:
Maybe we should append the baseline package version to left name?

Reply:
From a log perspective that information can be helpful, from a suppression perspective you don't want to duplicate that information in a bunch of places.  Are the two decoupled today?  Another option is to qualify left with something like "baseline"

Reply:
> From a log perspective that information can be helpful, from a suppression perspective you don't want to duplicate that information in a bunch of places. Are the two decoupled today? Another option is to qualify left with something like "baseline"

There is a flag in the suppression to tell if it only applies to a baseline error, If you were to baseline this error, you would do it like this:

```<Suppression>
   <DiagnosticID>CP0002</DiagnosticID>
   <Target>M:PackageValidationTestProject.Program.SomeApiNotInLatestVersion</Target>
   <IsBaselineSuppression>true</IsBaselineSuppression>
   <Left>lib/netstandard2.0/PackageValidationTestProject.dll</Left>
   <Right>lib/netstandard2.0/PackageValidationTestProject.dll</Right>
</Suppression>
```

You can even set a "left" name with spaces if we wanted, something like `lib/netstandard2.0/PackageValidationTestProject.dll on baseline`

Reply:
> Maybe we should append the baseline package version to left name?

we will need to do that on the package validation side i think, maybe pass a prefix

Reply:
how about 
```
error CP0002: Member 'PackageValidationTestProject.Program.SomeApiNotInLatestVersion()' exists on [Baseline] lib/netstandard2.0/PackageValidationTestProject.dll but not on lib/netstandard2.0/PackageValidationTestProject.dll
```
does that look fine ?

Reply:
That looks good to me. We could maybe have an informational header as well rather than introducing a prefix. 

@ericstj @terrajobst thoughts?